drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -227,14 +213,14 @@ def get_catalog(ra, dec, sr=0.1, fmt='CSV', catalog='GSC241'):
     fmt : str, optional
         Format of output catalog to be returned.  Options are determined by
         web-service, and currently include (Default: CSV):
-            VOTABLE(default) | HTML | KML | CSV | TSV | JSON | TEXT
+        VOTABLE(default) | HTML | KML | CSV | TSV | JSON | TEXT
 
     catalog : str, optional
         Name of catalog to query, as defined by web-service.  Default: 'GSC241'
 
```

Comment:
How does the `fmt` string parameter reconcile against the `csv` output parameter?    The implication from the output is there really is not a choice.

Reply:
This function does not need the `fmt` parameter at all, since this function will ALWAYS return a CSV dict reader object.  So, this has been removed.